airflow/api_internal/internal_api_call.py

```diff
@@ -39,6 +40,12 @@
 
 logger = logging.getLogger(__name__)
 
+class AirflowHttpException(AirflowException):
+    """Raise when there is a problem during an http request."""
```

Comment:
Maybe some re-phrasing to make it clear (so that the docstring has a benefit over the pure class name)
```suggestion
    """Raise when there is a problem during an http request on the internal API decorator."""
```